core/cas-server-core-tickets-api/src/main/java/org/apereo/cas/ticket/TicketGrantingTicketImpl.java

```diff
@@ -71,6 +72,12 @@ public class TicketGrantingTicketImpl extends AbstractTicket implements Authenti
      */
     private Set<String> descendantTickets = new HashSet<>(0);
 
+    /**
+     * Another identifier linked to this ticket.
+     */
+    @Setter
+    private String linkedIdentifier;
```

Comment:
I hesitated on this: this `linkedIdentifier` is a generic concept to be able to link the TGT to another identifier.
Currently, it is used for authn delegation.
I could have created a `DelegatedTicketGrantingTicketImpl` with that information but I feared side-effects and complexity with a new ticket type.